assets/js/script.js

```diff
@@ -4,6 +4,11 @@
   // Preloader js    
   $(window).on('load', function () {
     $('.preloader').fadeOut(100);
+    if(document.getElementById('calendar-container')){
+      const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone 
+      const html = `<iframe src="https://calendar.google.com/calendar/embed?src=c_62694f414055ac569e5cb12dafbb0890ca22f3640b177a4b10b53171fbc9bdd4%40group.calendar.google.com&ctz=${timezone}" style=" border:0 " width="800" height="600" frameborder="0" scrolling="no"></iframe>`
+      document.getElementById('calendar-container').innerHTML = html;
```

Comment:
A little more efficient to save the reference to the element instead of getting it again, but not a huge deal.